mne/evoked.py

```diff
@@ -402,10 +402,11 @@ def plot(self, picks=None, exclude='bads', unit=True, show=True, ylim=None,
             channel traces will not be shown.
         window_title : str | None
             The title to put at the top of the figure window.
-        spatial_colors : bool
-            Color code lines by mapping physical sensor coordinates into color
-            values. Spatially similar channels will have similar colors.
-            Bad channels will be dotted.
+        spatial_colors : None | 'head' | 'skirt' | 'notopo'
```

Comment:
honestly let's keep the API simple and address the 90% of use cases.

otherwise we'll end up with one option per paper that uses MNE.

personally I have branches with tiny commits for some of my papers and I
don't push this in MNE code base exactly for this reason. So now I think
that passing something like skirt / head is an API overfit !